proposals/0019-graphql.md

```diff
@@ -0,0 +1,130 @@
+# Solution name
+
+* Proposal: [SSWG-0019](0019-graphql.md)
+* Authors: [Jay Herron](https://github.com/NeedleInAJayStack)
+* Review Manager: TBD
+* Status: **Implemented**
+* Implementation: [GraphQL](https://github.com/GraphQLSwift/GraphQL)
```

Comment:
Think we could include the https://github.com/GraphQLSwift/Graphiti in the pitch as well? They seem to go very much hand in hand; would the authors and all the other criteria be met by this equally easily??

Reply:
Sure. I've made a follow-up commit that adds Graphiti into the proposal.